src/addRemove.js

```diff
@@ -15,12 +16,45 @@ function createTask(name) {
 function addTask(e) {
   e.preventDefault();
   const getTaskName = newTaskInput.value;
-  if (getTaskName === null || getTaskName === '');
   const task = createTask(getTaskName);
   newTaskInput.value = null;
   tasks.push(task);
-  save();
-  render();
+  save(tasks);
+  render(tasks);
+  document.location.reload();
 }
 
-export { newTaskForm, addTask };
+function deleteItem() {
+  const listItems = document.querySelectorAll('li');
+  listItems.forEach((item, index) => {
```

Comment:
- [ ] I would recommend using `es6` syntax as it is a project requirement which helps you in writing more clean code using ES6 syntax.